src/XrdHttp/XrdHttpProtocol.cc

```diff
@@ -2750,6 +2770,18 @@ int XrdHttpProtocol::xsecxtractor(XrdOucStream& Config) {
   return 0;
 }
 
+int XrdHttpProtocol::xcors(XrdOucStream& Config) {
+  char * val;
+  // Get the path
+  val = Config.GetWord();
+  if (!val || !val[0]) {
+    eDest.Emsg("Config", "No CORS plugin specified.");
+    return 1;
+  }
+  strlcpy(xrdcorsLibPath, val, sizeof(xrdcorsLibPath));
```

Comment:
Assuming the destination is a pointer, all you need to do is a) if not null, free it, b) do a simple strdup to set the pointer. If you prefer to use std::string then just assign it.